designer/client/components/Fee/FeeItems.jsx

```diff
@@ -131,9 +147,32 @@ export class FeeItems extends React.Component {
       });
   };
 
+  itemHasStaticQuantity = (item) => {
+    console.log("item multiplier: ", item.multiplier);
+    return !!parseInt(item.multiplier);
```

Comment:
why `!!`? What are you checking for here? That the value is falsey? (either 0, undefined or null, NaN etc)

FWIW - always avoid using `!!` if you know your value will be a boolean (in which case it doesn't make sense to "not not" it.. it would just convert it back to the original value). For thee same reason, avoid using `!!` if you know the value can be 0 or 1. 

0 is a falsey value, but is a valid number. I think probably it still "works" for what you're intending, but ultimately safe and could trip you up in future. using `!!` on 0  will give you false.

I would also write unit tests any time I need to use `!!` and my values are possibly falsey

```
> !parseInt("0")
true
> !parseInt("1")
false
> !parseInt("not a number");
true
> !parseInt("5")
false
> !!parseInt("0");
false
> !!parseInt("1");
true
> !!parseInt("not a number");
false
> !!parseInt("5");
true

```